scripts/legacy_expression_tool/README.md

```diff
@@ -82,24 +82,24 @@ python legacy_expression_tool.py \
-| `--service-path`       | The fiware service path filter to replace the expressions                                     | All subservices              | No        |
-| `--deviceid`           | The device id filter to replace the expressions                                               | All devices                  | No        |
-| `--entitytype`         | The entity type filter to replace the expressions                                             | All entity types             | No        |
-| `--regexservice`       | The fiware service regex filter to replace the expressions                                    | All subservices              | No        |
-| `--regexservicepath`   | The fiware service path regex filter to replace the expressions                               | All subservices              | No        |
-| `--regexdeviceid`      | The device id regex filter to replace the expressions                                         | All devices                  | No        |
-| `--regexentitytype`    | The entity type regex filter to replace the expressions                                       | All entity types             | No        |
+| Argument               | Description                                                                                              | Default value                | Mandatory |
+| ---------------------- | -------------------------------------------------------------------------------------------------------- | ---------------------------- | --------- |
+| `--mongouri`           | The MongoDB URI to connect to                                                                            | `mongodb://localhost:27017/` | No        |
+| `--database`           | The database name to replace the expressions                                                             | NA                           | Yes       |
+| `--collection`         | The collection name to replace the expressions                                                           | NA                           | Yes       |
+| `--translation`        | The translation dictionary file to replace the expressions                                               | `translation.json`           | No        |
+| `--debug`              | Enable debug mode                                                                                        | `False`                      | No        |
+| `--commit`             | Commit the changes to the database                                                                       | `False`                      | No        |
+| `--expressionlanguage` | What to do with the expression language field. Possibles values: `delete` (to delete the field), `ignore` (to leave the field untouched), `jexl` (to force to jexl on all groups/devices that match the conditions) or `jexlall` (like `jexl` but additionally including all groups/devices that have `expressionLanguage` field to the query) | `ignore`                     | No        |
```

Comment:
```suggestion
| `--expressionlanguage` | What to do with the expression language field. Possibles values: `delete` (to delete the `expressionLanguage` field), `ignore` (to leave the field untouched, **FIXME** don't use, it can leave documents inconsistent), `jexl` (to change expressions to JEXL and in addition to change `expressionLanguage` field to `jexl` when the group/device uses legacy in some of the expression-related field, no matter if the group/devices have previously the `expresionLanguage` field or not) or `jexlall` (like `jexl` option, but also including groups/devices that doesn't use JEXL i any expression-related field but have `expresionLanguage` field defined, no matter its value) | `ignore`                     | No        |
```